tools/build/Makefile

```diff
@@ -25,9 +25,16 @@ PYTHON_DIR=$(DSTDIR)/lib/python$(PY_VERSION)/site-packages/
 
 all: environment template enclavekey packages
 
+no_venv: template enclavekey packages_no_venv
```

Comment:
I would add a 1 line comment that this is the same as `make all` without setting up a Python virtual environment.

Reply:
You could consider conditionally building a target and it's dependencies instead of adding multiple targets like no_env, clean_pip_packages, packages_no_env
for example,
ALL_TARGETS = template enclavekey packages
ifeq ($(STAND_ALONE_BUILD),1)
    ALL_TARGETS += environment
endif
all: $(ALL_TARGETS)

clean :
     ifeq ($(STAND_ALONE_BUILD),1)
          rm -rf /usr/lib/python3/dist-packages/tcf*
     else
          rm -rf $(DSTDIR)
      endif
      $(abspath $(SRCDIR)/../clean.sh)

Reply:
I moved installation instruction of pip packages to pre-requisites section in documentation and Make file will not download any packages. In the cleanup section, source code related packages are deleted either from standard location as in the case of docker or virtual environment in the standalone mode.

Reply:
Ok. Let's not call standalone or docker now onwards. Let's call it install and uninstall irrespective of which machine it is :)